### PR TITLE
Add Ozone UFS documentation

### PR DESCRIPTION
**Pull request by maobaolong**

### CODE REVIEW

**Comment by maobaolong** [comment]
@ZacBlanco Would you like to review this documents, any suggestion will be appreciated.

File: docs/en/ufs/Ozone.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+---
+layout: global
+title: Ozone
+nickname: Ozone
+group: Storage Integrations
+priority: 10
+---
+
+
+This guide describes how to configure [Ozone](http://hadoop.apache.org/ozone) as Alluxio's under storage system.
```
[the remark this answers is not in the document]
Comment (maobaolong) [reply docs/en/ufs/Ozone.md]: @ZacBlanco  Done.

File: docs/en/ufs/Ozone.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+Ozone can function effectively in containerized environments such as Kubernetes and YARN.
+
+## Prerequisites
+
+To run an Alluxio cluster on a set of machines, you must deploy Alluxio binaries to each of these
+machines. You can either [download the precompiled binaries directly]({{ '/en/deploy/Running-Alluxio-Locally.html' | relativize_url }})
+with the correct Hadoop version (recommended), or 
+[compile the binaries from Alluxio source code]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}) (for advanced users).
+
+In preparation for using Ozone with Alluxio, follow the [Ozone On Premise Installation](http://hadoop.apache.org/ozone/docs/0.5.0-beta/start/onprem.html)
```
[the remark this answers is not in the document]
Comment (maobaolong) [reply docs/en/ufs/Ozone.md]: @ZacBlanco Done.

File: docs/en/ufs/Ozone.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+to install a Ozone cluster, and follow the [Command Line Interface](http://hadoop.apache.org/ozone/docs/0.5.0-beta/shell.html)
```
[the remark this answers is not in the document]
Comment (maobaolong) [reply docs/en/ufs/Ozone.md]: @ZacBlanco Done.

File: docs/en/ufs/Ozone.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+to create volume and bucket for Ozone cluster.
```
[the remark this answers is not in the document]
Comment (maobaolong) [reply docs/en/ufs/Ozone.md]: @ZacBlanco  I agree to link to the specific document. Done.

**Comment by maobaolong** [comment]
@ZacBlanco Thank you for your reviewing and suggestion. Please take another look.

**Comment by ZacBlanco** [comment]
alluxio-bot, merge this please

**Pull request merged**